### ISSUE TITLE
What if MNet keypair is changed

### ISSUE THREAD

**Issue by mudrd8mz** [vfremaux/moodle-auth_ticket#5]
It seems to me that the ticket encoding / decoding relies on the keypair obtained via `mnet_get_keypair()`. If I recall correctly, MNet changes the keys regularly. Does it mean that each such change invalidates all issued tickets?

**Comment by vfremaux** [comment]
yes it may actually. Operationally we use to push the duration of the mnet key a bit upper than 28 days as in standard. The issue is that asking for a separate key adds in config complexity ( i mean in an overall sense if all plugins needing ssl security ask for their own key). Comment welcome on a cleverer alternative.
